requirements.txt

```diff
@@ -25,14 +25,14 @@
 # argparse>=1.4.0
 # argparse - builtin - PSF license
 # setuptools - MIT license
-setuptools>=45.0
+setuptools>=75.0
```

Comment:
LGTM! We are now dropping support for pre-3.9 (as part of #134) Makes sense to bump versions too.